include/nbl/asset/IAssetManager.h

```diff
@@ -326,7 +326,7 @@ class IAssetManager : public core::IReferenceCounted, public core::QuitSignallin
 
             system::ISystem::future_t<core::smart_refctd_ptr<system::IFile>> future;
             bool validInput = m_system->createFile(future, _filePath, system::IFile::ECF_READ);
```

Comment:
will createFile return false if the file path is too long?

Reply:
yes

Reply:
how long is too long?

Reply:
4096 characters. But this is not OS path length check, it's just the request params have a field `char filename[MAX_FILENAME_LENGTH]` so there is a check if the path will fit into that array.